packages/backend/src/db/db.module.ts

```diff
@@ -12,13 +14,15 @@ import { Bill, BillSchema } from './bill/bill.schema';
     MongooseModule.forFeature([{ name: User.name, schema: UserSchema }]),
     MongooseModule.forFeature([{ name: House.name, schema: HouseSchema }]),
     MongooseModule.forFeature([{ name: Bill.name, schema: BillSchema }]),
+    MongooseModule.forFeature([{ name: Note.name, schema: NoteSchema }]),
   ],
-  providers: [UserStoreService, HouseStoreService, BillStoreService],
+  providers: [UserStoreService, HouseStoreService, NoteStoreService],
```

Comment:
This shouldnt be removing the bill store service 
```suggestion
  providers: [UserStoreService, HouseStoreService, BillStoreService, NoteStoreService],
```

Reply:
careless merging

Reply:
😢